phing/tasks/setup.xml

```diff
@@ -248,13 +248,17 @@
           <param>drush</param>
         </drush>
         <!-- Revert all features. -->
-        <drush command="fra" assume="yes" alias="${drush.alias}">
-          <option name="bundle">${cm.features.bundle}</option>
-        </drush>
+        <foreach list="${cm.features.bundle}" param="bundle" target="setup:update:features-import" />
```

Comment:
@grasmash I tested it with yml formatted as such:

```yaml
cm:
  features:
    bundle: my_feature_bundle
```
This worked as expected, feeding into the foreach as an array with a single value.

I think 'bundles' is clearer myself but I wanted to not breaking existing project.yml.